src/pybind/mgr/dashboard/frontend/src/app/shared/enum/dashboard-promqls.enum.ts

```diff
@@ -8,3 +8,10 @@ export enum Promqls {
   WRITECLIENTTHROUGHPUT = 'sum(rate(ceph_pool_wr_bytes[$interval]))',
   RECOVERYBYTES = 'sum(rate(ceph_osd_recovery_bytes[$interval]))'
 }
+
+export enum RgwPromqls {
+  RGW_REQUEST_PER_SECOND = 'sum(rate(ceph_rgw_req[$interval]))',
+  AVG_GET_LATENCY = 'sum(rate(ceph_rgw_get_initial_lat_sum[$interval])) / sum(rate(ceph_rgw_get_initial_lat_count[$interval]))',
+  AVG_PUT_LATENCY = 'sum(rate(ceph_rgw_put_initial_lat_sum[$interval])) / sum(rate(ceph_rgw_put_initial_lat_count[$interval]))',
+  BANDWIDTH = 'sum(rate(ceph_rgw_get_b[$interval])) %2B sum(rate(ceph_rgw_put_b[$interval]))'
```

Comment:
Don't trade off readibility by manually encoding symbols. Angular provides several methods for encoding URL params:
```suggestion
  BANDWIDTH = 'sum(rate(ceph_rgw_get_b[$interval])) + sum(rate(ceph_rgw_put_b[$interval]))'
```

Reply:
Sure, will change this